### ISSUE TITLE
Request entity too large Error

### ISSUE THREAD

**Issue by Xotabu4** [kochetkov-ma/allure-server#42]
Getting error on attempt to upload zip file. 

Is there a way to configure max file size? 

![image](https://user-images.githubusercontent.com/3033972/146180250-f0fe3931-f919-40f5-b001-82027aab7633.png)

**Comment by kochetkov-ma** [comment]
There are two parameters in the Spring settings:
```
spring:
  servlet:
    multipart.max-file-size: 100MB
    multipart.max-request-size: 100MB
```

It can be specified via environment variables when you start the container: 
`spring_servlet_multipart_maxRequestSize=1000MB`
`spring_servlet_multipart_maxFileSize=1000MB`

Try this ...

**Comment by Xotabu4** [comment]
Thanks, will try tomorrow

**Comment by Xotabu4** [comment]
Nope, uploading fails even for 1mb zip file. Restarting k8s pod didn't help

I don't see any errors in logs. Maybe i have been hacked because of https://www.zdnet.com/article/log4j-zero-day-flaw-what-you-need-to-know-and-how-to-protect-yourself/ ?

**Comment by kochetkov-ma** [comment]
Already solved such a problem in  #16 . 
Strange that it does not help, but it was not in Kubernetes. 
How do you deploy? Through the chart from the project? 
In Kubernetes you still need to enable the setting for nginx ingress : 
```
  annotations:
    kubernetes.io/ingress.class: nginx
    ## deprecated
    nginx.ingress.kubernetes.io/proxy-body-size: "100m"
    ## deprecated
    nginx.ingress.kubernetes.io/client-max-body-size: "100m"
    ## new https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/
    nginx.org/client-max-body-size: "100m"
```
I tested all of the settings on Yandex Cloud - it was ok.

Provide me your deployment configuration or helm chart

**Comment by Xotabu4** [comment]
@kochetkov-ma We synced helm-chart. Seems like latest changes with ingress in cluster something bad happened

Works fine now! Thanks!
